src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -3502,7 +3495,7 @@ void writeByte(byte value) throws SQLServerException {
                 if (dataIsLoggable)
                     logBuffer.put(value);
                 else
-                    ((Buffer) logBuffer).position(((Buffer) logBuffer).position() + 1);
+                    logBuffer.position(logBuffer.position() + 1);
```

Comment:
Are braces needed for the if/else? I can't recall if there were checks that complained about missing braces.

Reply:
Not needed, but I believe that's our code style. I'm surprised SonarQube didn't complain about it.